tern/formats/default/generator.py

```diff
@@ -9,7 +9,7 @@
 
 import logging
 
-from tern.report import formats
+from tern.report import formats,content
```

Comment:
content is already imported a few lines below :)

Reply:
yeah i found that while fixing some linting issues

Reply:
Anything other than that
..?

Reply:
I am still reviewing so please hold off on pushing your changes just yet :)